ts/features/bonus/cdc/screens/CdcBonusRequestSelectResidence.tsx

```diff
@@ -20,26 +20,36 @@ import {
   cancelButtonProps,
   confirmButtonProps
 } from "../../bonusVacanze/components/buttons/ButtonConfigurations";
+import { cdcSelectedBonusSelector } from "../store/reducers/cdcBonusRequest";
+import { useIOSelector } from "../../../../store/hooks";
+import { H3 } from "../../../../components/core/typography/H3";
+import BonusIcon from "../../../../../img/features/cdc/bonus.svg";
 
-const getCheckResidencyItems = (): ReadonlyArray<RadioItem<boolean>> => [
+const getCheckResidencyItems = (): ReadonlyArray<RadioItem<residentChoice>> => [
   {
     body: I18n.t("bonus.cdc.bonusRequest.selectResidence.items.residesInItaly"),
-    id: true
+    id: "residentInItaly"
   },
   {
     body: I18n.t("bonus.cdc.bonusRequest.selectResidence.items.residesAbroad"),
-    id: false
+    id: "residentAbroad"
   }
 ];
 
+type residentChoice = "residentAbroad" | "residentInItaly";
```

Comment:
```suggestion
type residentChoice = "italy" | "notItaly";
```
these are the value close to my mental model
feel free to keep or skip

Reply:
I don't have a strong opinion, updated in [2c95dea](https://github.com/pagopa/io-app/pull/3949/commits/2c95deadcac4f0326f1895bb67a2e776e9f15369)